main/settings.py

```diff
@@ -1020,7 +1020,9 @@ def get_all_config_keys():
 
 # Relative URL to be used by Djoser for the link in the password reset email
 # (see: http://djoser.readthedocs.io/en/stable/settings.html#password-reset-confirm-url)
-PASSWORD_RESET_CONFIRM_URL = "password_reset/confirm/{uid}/{token}/"
+PASSWORD_RESET_CONFIRM_URL = (
```

Comment:
Or we can keep them for consistent code style?

Reply:
We can consider the string we are formatting is going to work the way it used to. Sometimes URLs & regexes break after the automated tools format them. Since we don't have 100% coverage so we should be careful about it. This was the reason I was inclined to ignore this with a warning instead of this formatting.